pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -336,6 +332,9 @@ func (plugin *kubenetNetworkPlugin) Capabilities() utilsets.Int {
 // setup sets up networking through CNI using the given ns/name and sandbox ID.
 func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kubecontainer.ContainerID, annotations map[string]string) error {
 	var ipv4, ipv6 net.IP
+	var podGateways []net.IP
+	var podCIDRs []net.IPNet
```

Comment:
nit, we are going to have a method var named `plugin.podCIDR`s and this local var `podCIDRs` , can it be misleading?